assets/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "newspack-components",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
Why the version bump here? I don't see any changes in the components?

Reply:
That was to publish the latest changes on npm. We update components all the time, but releasing `newspack-components` on npm is done manually.